recipes/yaclib/all/conandata.yml

```diff
@@ -0,0 +1,12 @@
+sources:
+  '2022.10.31':
+    url: 'https://github.com/YACLib/YACLib/archive/refs/tags/v2022.10.31.tar.gz'
+    sha256: '81761b1c8e53e6eaeb36fa00183cae66068b85d24c910c0584d0b29b371e143c'
+patches:
+  '2022.10.31':
+    - patch_file: 'patches/0001-Set-17-as-default-or-20-if-CORO-specified.patch'
+      patch_description: 'Set 17 standard if not stated otherwise'
+      patch_type: 'conan'
+    - patch_file: 'patches/0002-Add-install-commands.patch'
+      patch_description: 'Add install commands'
+      patch_type: 'conan'
```

Comment:
This should not be a patch, the could be logic in the recipe to copy the directories. 

We'd like to have the lest amount of patches possible, I think this is a case where we could trim back